docs/glossary.md

```diff
@@ -0,0 +1,60 @@
+
+| Term | Definition |
+|----|----|
+| Riskline  | Amadeus' travel restrictions data provider.    |
+
+### [Air](https://developers.amadeus.com/self-service/category/air) and [Trip](https://developers.amadeus.com/self-service/category/trip)
+
+| Term | Definition |
+|----|----|
+| Additional Baggage | Luggage beyond the standard allowance provided by an airline, subject to additional fees. |
+| Aircraft Code         | [IATA aircraft code](http://www.flugzeuginfo.net/table_accodes_iata_en.php).              |
+| Airline Code          | Airline code following IATA or ICAO standard - e.g. 1X; AF or ESY.  |  
+| Airline consolidators | Wholesalers of air tickets that usually partner with airlines to get negotiated rates for air tickets, and then resell the air tickets to travel agents or consumers. |
+| Amadeus Office ID | An identification number assigned to travel agencies to access Amadeus system and book reservations. |
+| Amenities | Additional services or features offered to enhance the experience of the passengers, such as food, entertainment, Wi-Fi, extra legroom, baggage allowance, frequent flyer programs, and lounge access. They can vary depending on the class of service and the airline/train company. |
+| Baggage allowance | The amount of luggage that a passenger is allowed to carry on a flight without additional charges. |
+| Booking | The process of reserving a seat on a flight or a room in a hotel. |
+| Cabin | The section of an aircraft or train where passengers sit during their trip. It is divided into different classes, such as first class, business class, and economy class, each one with different amenities and prices. |
+| Commission | Fee paid to intermediaries for booking travel-related services, usually a percentage of the total cost. |
+| Carrier Code          | 2 to 3-character IATA carrier code (IATA table codes).  |
+| Country Code          | Country code following ISO 3166 Alpha-2 standard.         |
+| Direct flight | A flight that goes from one destination to another without any stops in between. |
+| Fare | The price of a ticket for a particular flight or travel itinerary. |
+| Fare Rules | The terms and conditions that apply to a specific airline ticket or fare, including restrictions and information on refunds, cancellations, changes, baggage, seat assignments, upgrades, and frequent flyer programs. |
+| Flight Order Id       | Unique identifier returned by the [Flight Create Orders API](https://developers.amadeus.com/self-service/category/air/api-doc/flight-create-orders). |
+| GDS (Global Distribution System) | A computerized system used by travel agents and airlines to search for and book flights, hotels, rental cars, and other travel-related services |
+| IATA | [International Air Transport Association](www.iata.org) |
+| IATA Code             | [Code](https://www.iata.org/en/publications/directories/code-search/) used by IATA to identify locations, airlines and aircraft.     |
```

Comment:
shall we mention about our Airport & city search API as the API is returning the IATA code instead of IATA website